### ISSUE TITLE
Most ideal normalization space for expression data?

### ISSUE THREAD

**Issue by rgalvin516** [CRI-iAtlas/ImmuneSubtypeClassifier#9]
Hello,

Firstly, thank you for compiling this resource for the public. The whole work is an incredible inspiration to me and others. I have been using gene expression data and exploring these immune subtypes. I noticed that there is a significant difference in best immune subtype calls depending on whether I run RSEM expected counts vs TPM gene expression data. The TPM data results in a much higher proportion of C4 in my data. I was wondering if in theory, one or the other would be more "valid"?

Thanks for your time

**Comment by Gibbsdavidl** [comment]
Hi, thanks for writing. Glad to hear it's hopefully useful.

The way I built the classifier, the method of gene quantization should not matter.  TPM, RPKM, etc, should all be identical.

Now, I am going to guess here. But, while the classifier requires no normalization, it is actually sensitive to, and harmed by, gene-wise normalization.  What I mean is: If you take a matrix of gene expression with genes in columns and samples in rows, and normalize the gene expression across samples (median scaling for example), the classification is not valid.  And as I've seen it, often produces C4 subtype calls.

This is because, the classification is now based on comparing pairs of genes and producing binary features. Internally to a sample, if gene_x > gene_y the feature=1 else feature=0.  Now, if you normalize genes across samples, the gene-gene comparison has been changed as the values have been modified.

Does this help at all?  I've been working (not much recently) on a paper that explains the classifier. It's here:   
https://www.biorxiv.org/content/10.1101/2020.01.17.910950v1
